phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/controllers/config/CloseChannelsConfigurationController.kt

```diff
@@ -100,13 +105,16 @@ class AppCloseChannelsConfigurationController(
                     val wallet = walletManager.wallet.value!!
                     val address = wallet.onchainAddress(path)
```

Comment:
We should rather use `Peer.finalAddress` (from lightning-kmp) instead of computing the address ourselves. This would also allow us to remove the `Wallet.onchainAddress` method (and the tests)

@pm47 can you confirm this is the correct address? Btw this field does not seem to be used (nor tested) in lightning-kmp.

Reply:
Fixed in e2f5762

(It checked, and `Peer.finalAddress` does generate the same address as the previous code. So it does look like this is the proper replacement.)

Reply:
> @pm47 can you confirm this is the correct address for force-close? Btw this field does not seem to be used (nor tested) in lightning-kmp.

Yes it's the correct address, you can use it. This are is in-progress though, so the api may change.

Reply:
Great! can you also remove the `Wallet.onchainAddress` method that should now be unused, as well as the relevant tests in `WalletTest`?

Reply:
Done in [3576d04](https://github.com/ACINQ/phoenix/pull/303/commits/3576d043b4996fec108eec70d1fec824663e8f27)